reana_helloworld/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM condaforge/mambaforge:latest
+LABEL io.github.snakemake.containerized="true"
+LABEL io.github.snakemake.conda_env_hash="a27eb1bd907da7d709dcbe093527f3ae2ce3ac8b4a9f4ec63cc1a34b4e1f4dc0"
```

Comment:
```suggestion
```

Unless Snakemake explicitly uses these `LABEL`s to validate or check information I would remove them.